doc/api/zlib.md

```diff
@@ -89,17 +122,25 @@ request.on('response', (response) => {
 
   switch (response.headers['content-encoding']) {
     case 'br':
-      response.pipe(zlib.createBrotliDecompress()).pipe(output);
+      pipeline(response, zlib.createBrotliDecompress(), output, (err) => {
+        // Handle error.
+      });
       break;
     // Or, just use zlib.createUnzip() to handle both of the following cases:
     case 'gzip':
-      response.pipe(zlib.createGunzip()).pipe(output);
+      pipeline(response, zlib.createGunzip(), output, (err) => {
+        // Handle error.
+      });
       break;
     case 'deflate':
-      response.pipe(zlib.createInflate()).pipe(output);
+      pipeline(response, zlib.createInflate(), outout, (err) => {
+        // Handle error.
+      });
       break;
     default:
-      response.pipe(output);
+      pipeline(response, output, (err) => {
+        // Handle error.
+      });
       break;
```

Comment:
Please handle the error in these cases. It's an http server, so we know how to handle it: return a 500.

Reply:
This is the client side. Will handle by printing error message and exiting the process.